pkg/controller/garbagecollector/graph_builder.go

```diff
@@ -288,15 +288,18 @@ func (gb *GraphBuilder) IsSynced() bool {
 	defer gb.monitorLock.Unlock()
 
 	if len(gb.monitors) == 0 {
+		glog.V(2).Info("garbage controller monitor not synced: no monitors")
```

Comment:
Just curious, why isn't the set of all monitors considered synced? It makes more sense to me to say that it is synced, because all 0 of the monitors are synced.

Reply:
I'd be open to revisiting this, but wanted to keep current behavior in this PR